identity/conf/routes

```diff
@@ -62,6 +62,7 @@ POST        /privacy/edit-ajax                      controllers.EditProfileContr
 GET         /email-prefs                            controllers.EditProfileController.displayEmailPrefsForm(consentsUpdated: Boolean ?= false, consentHint: Option[String])
 POST        /email-prefs                            controllers.EditProfileController.saveEmailPreferencesAjax
 
-GET         /consent                                controllers.EditProfileController.displayConsentJourneyForm(journey: Option[String], consentHint: Option[String], newsletterHint: Option[String])
-POST        /complete-consent                         controllers.EditProfileController.submitRepermissionedFlag
+GET         /consents                               controllers.EditProfileController.displayConsentJourneyForm(journey = "consent", consentHint: Option[String], newsletterHint: Option[String])
+GET         /consents/:journey                      controllers.EditProfileController.displayConsentJourneyForm(journey: String, consentHint: Option[String], newsletterHint: Option[String])
+POST        /complete-consent                       controllers.EditProfileController.submitRepermissionedFlag
```

Comment:
do we need to pluralise this too? `complete-consents`

Reply:
I think we should yep